### PR TITLE
Fix isWritable incorrect problem.

### PR DESCRIPTION
**Pull request by horizonzy**
### Motivation
The `isWritable` should be false in some places, correct it.

### CODE REVIEW

**Comment by horizonzy** [comment]
@dlg99 cc.

**Review by hangc0276 — APPROVED** [review]

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PerChannelBookieClient.java [hunk 1]
```diff
@@ -1097,7 +1097,7 @@ private void closeInternal(boolean permanent, boolean wait) {
             }
             toClose = channel;
             channel = null;
-            makeWritable();
+            makeUnWritable();
```
Comment (zymap) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PerChannelBookieClient.java:1100]: The writable should be a reflection value of the Channel's isWritable. So we make it default to true. So it should always be true except when the channel is non-writable.

**Review by zymap — COMMENTED** [review]
@michaeljmarshall Could you please take a look at this change?

**Review by dlg99 — CHANGES_REQUESTED** [review]
I see couple of problems with the PR:

1. It claims the X is a problem, fixes X, does not explain what's the problem, what it affects etc. The PR changes existing test to match but does not add a test reproducing the problem it was actually fixing. I don't understand what led you to this change, did you just looked at he code and decided to modify it or there is actual problem you have encountered?

2. I don't remember the call chain that well, but II think the idea is that PCBC with disconnected channel must be writable. Otherwise whoever needs to write to the PCBC may decide to skip at non-writable instead of just writing and triggering reconnect.

**Pull request closed without merging**